de-lib/src/main/java/org/iplantc/de/admin/apps/client/presenter/callbacks/AppStatsSearchCallback.java

```diff
@@ -0,0 +1,14 @@
+package org.iplantc.de.admin.apps.client.presenter.callbacks;
+
+import com.google.web.bindery.autobean.shared.Splittable;
+
+import jsinterop.annotations.JsFunction;
+
+/**
+ * Created by sriram on 3/27/18.
+ */
+@JsFunction
+@SuppressWarnings("unusable-by-js")
```

Comment:
So what's the warning here? It thinks `Splittable[]` is unusable?

Maybe if we pass the service results as a single `Splittable results` with the array in `results.apps`, then it will get rid of the warning?